Sources/MongoSwift/MongoConnectionString.swift

```diff
@@ -420,7 +430,6 @@ public struct MongoConnectionString: Codable, LosslessStringConvertible {
             throw MongoError.InvalidArgumentError(message: "Invalid connection string")
         }
         let identifiersAndOptions = schemeAndRest[1].components(separatedBy: "/")
-        // TODO: SWIFT-1174: handle unescaped slashes in unix domain sockets.
```

Comment:
Not sure why this was here; the spec [indicates](https://github.com/mongodb/specifications/blob/master/source/connection-string/connection-string-spec.rst#host) that hosts cannot contain unescaped slashes and there are no tests that allow for this behavior.